README.md

```diff
@@ -20,27 +18,155 @@
 require('apostrophe')({
   shortName: 'my-project',
   modules: {
-    '@apostrophecms/module-template': {}
+    '@apostrophecms/security-headers': {}
   }
 });
 ```
 
-### Additional usage sections
+The headers can be overridden by setting them as options to the module:
+
+```javascript
+// in app.js
+modules: {
+  'apostrophe-security-headers': {
+    'X-Frame-Options': 'DENY'
+  }
+}
+```
+
+You can also disable a header entirely by setting the option to `false`.
+
+The `Content-Security-Policy` header is more complex. The default response for it is the result of merging together options for individual use cases as shown below. However you may also simply set a string for it to override all of that.
+
+### Default behavior
+
+Here are the headers that are sent by default, with their default values:
+
+```javascript
+  // 1 year. Do not include subdomains as they could be unrelated sites
+  'Strict-Transport-Security': 'max-age=31536000',
+  // You may also set to DENY, however future Apostrophe modules may use
+  // iframes to present previews etc.
+  'X-Frame-Options': 'SAMEORIGIN',
+  // If you have issues with broken images etc., make sure content type
+  // configuration is correct for your production server
+  'X-Content-Type-Options': 'nosniff',
+  // Very new. Used to entirely disable browser features like geolocation per host.
+  // Since we don't know what your site may need, we don't try to set this
+  // header by default (false means "don't send the header")
+  'Permissions-Policy': false,
+  // Don't send a "Referer" (sp) header unless the new URL shares the same
+  // origin. You can set this to `false` if you prefer cross-origin "Referer"
+  // headers be sent. Apostrophe does not rely on them
+  'Referrer-Policy': 'same-origin',
+  // `true` means it should be computed according to the rules below.
+  // You may also pass your own string, or `false` to not send this header.
+  // The `policies` option and all of its sub-options are ignored unless
+  // `Content-Security-Policy` is `true`.
+  'Content-Security-Policy': true,
+
+  // You do not have to copy and paste this entire example.
+  // The sub-options you specify for `policies` are intelligently merged
+  // with the defaults you see below. Any sub-option you specify explicitly at
+  // project level overrides all of the default settings shown below for that
+  // sub-option; you may set one to `false` to completely
+  // disable it. You may also introduce entirely new sub-options,
+  // which will also be honored.
+  //
+  // Policies of the same type from different sub-options are merged, with
+  // the largest set of keywords and hosts enabled. This is done because
+  // browsers do not support more than one style-src policy, for example, but
+  // do support specifying several hosts.
+  //
+  // Note the HOSTS wildcard which matches all expected hosts including CDN hosts
+  // and workflow hostnames.
+
+  policies: {
+    general: {
+      'default-src': `HOSTS`,
```

Comment:
A few of these `HOSTS` references are in backticks. Is that on purpose? This one and at least two below.